src/m365/teams/commands/chat/chat-get.ts

```diff
@@ -0,0 +1,191 @@
+import { AadUserConversationMember, Chat, ConversationMember } from '@microsoft/microsoft-graph-types';
+import { AxiosRequestConfig } from 'axios';
+import Auth from '../../../../Auth';
+import * as os from 'os';
+import request from '../../../../request';
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+import { validation } from '../../../../utils/validation';
+import { accessToken } from '../../../../utils/accessToken';
+import { odata } from '../../../../utils/odata';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id?: string;
+  participants?: string;
+  name?: string;
+}
+
+class TeamsChatGetCommand extends GraphCommand {
+  public get name(): string {
+    return commands.CHAT_GET;
+  }
+
+  public get description(): string {
+    return 'Get a chat conversation by id, participants or chat name';
```

Comment:
We should align the description of the documentation with that description.

Reply:
Hi @plamber ,

I checked what you mean.

```powershell
m365 teams chat get -p martin@blimped.nl,adelev@blimped.nl
```

This does not work because the comma is interpreted by powershell. The string value is now interpreted as two values and concatenated with a space.

==> I'l fix it by updating the `convertCommaSeparatedListToArray()` function to be able to split by space if necessary.

```powershell
m365 teams chat get --name Let's 
```
The single quote is causing this when posting to Microsoft Graph. In OData, single quotes need to be duplicated for the query to work. 

==> I'll fix it by replacing all `'` with `''`